src/test/java/io/openliberty/tools/intellij/it/UIBotTestUtils.java

```diff
@@ -744,7 +744,7 @@ public static void insertConfigIntoConfigFile(RemoteRobot remoteRobot, String fi
                 "Text " + configNameSnippet + " did not appear in the completion suggestion pop-up window",
```

Comment:
I think `configNameChooserSnippet` would be more valid for lines 743 and 744 instead of `configNameSnippet`. Although it is the combination we're looking for, I think the "chooser" is more of what we are looking for than the original snippet.